### ISSUE TITLE
[OTHER] Clean up repo

### ISSUE THREAD

**Issue by tommach591** [valhalla/web-app#143]
### What would you like to share?

I noticed that the files are a bit unorganized/inconsistent in my opinion and I would like to clean it up **during the GSoC 2023** alongside #68. (Not the red, not sure why those are there, could be VSCode.)

![Capture](https://user-images.githubusercontent.com/57111473/227033605-a9e73046-4925-4d1f-9420-a053d8eed78c.PNG)

I am suggesting to revise the files like so:

ComponentName - folder name
- ComponentName.jsx - the functional component 
- ComponentName.css - a css file to overwrite the parent css
- index.js - import ComponentName from ./ComponentName.jsx and export default ComponentName so files can just import as ./ComponentName instead of ./ComponentName/ComponentName
- other necessary js files/components - children components or js files used solely in ComponentName

This would make the repo easier to work with.

**Comment by nilsnolde** [comment]
Yeah absolutely! It's not very clean IMO either. We'll get to that during the project, thanks for raising!
